dotnet/Capstone/Controllers/BirdListController.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Capstone.DAO;
+using Capstone.Exceptions;
+using Capstone.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdListController: ControllerBase
+    {
+        private readonly IBirdListDao birdListDao;
+
+        public BirdListController(IBirdListDao birdListDao)
+        {
+            this.birdListDao = birdListDao;
+        }
+
+
+
+        [HttpGet("/lists/{id}")]
+        public BirdList getList(int id)
+        {
+            BirdList birdList = birdListDao.getList(id);
+
+            if (birdList == null)
+            {
+                Console.WriteLine("No bird matches this id");
+            }
+
+            return birdList;
+
+        }
+
+
+        [HttpGet("/lists")]
```

Comment:
This is called in the front end when displaying saved lists, so it should only retrieve lists for the logged in user.

Reply:
Done